src/mixins/render-vega-lite/Definitions/Encoding/Scale/Continuous/ContinuousScale.js

```diff
@@ -74,18 +74,26 @@ export default class ContinuousScale extends ScaleDefinitionObject {
     }
   }
 
+  // eslint-disable-next-line no-undef
+  static max_subdivisions = 25
```

Comment:
`25` is an arbitrary pick, but when talking about continuous scales, this should be more than plenty.

Reply:
Yup, more than enough with the colors we're sending over